docker/candlepin-base/cp-test

```diff
@@ -45,6 +45,13 @@ build_lint() {
 build_unittest() {
     UTCMD=$1
     if [ -f ../gradlew ]; then
+
+        # set project to run units from the candlepin-common project
+        cd ../common/
+        ../gradlew test
+
+        # set project back to server
+        cd ../server/
```

Comment:
Instead of doing `cd` everywhere, we should simplify things, and just run `.gradlew test` from the root directory.
So here this if block would become:
```
    if [ -f ./gradlew ]; then
        ./gradlew test
```
and later on (right before we run the unit tests), we don't cd into server/ , but cd into the root dir (so `cd $PROJECT_DIR` turns into `cd $CP_HOME`. And we only do a `cd $PROJECT_DIR` later, once the unit tests are finished and before we setup qpid

Reply:
Agreed, simplified the code.